packages/it-multipart/index.js

```diff
@@ -143,6 +198,7 @@ function waitForStreamToBeConsumed (stream) {
         return next
       } catch (err) {
         pending.reject(err)
+        throw err
```

Comment:
📝 TS rightly points out that without `throw` here iterator did not comply with `Iterator<T>` interface because  `next()` returns `{ done:boolean, value: T }|undefined` (`undefined` is returned because catch was swallowing an error). Added throw here so it type checks & I think that is also what code supposed to do.